packages/seccomp-tools/PKGBUILD

```diff
@@ -0,0 +1,78 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=seccomp-tools
+pkgver=v1.6.1.r36.g728ec2f
+ver="${pkgver%%.r*}" 
+pkgrel=1
+pkgdesc="Provide powerful tools for seccomp analysis."
+arch=('any')
```

Comment:
there is some C code built (native gem)) so it's arch dependant

Reply:
Fixed